code/modules/atmospherics/machinery/pipes/pipes.dm

```diff
@@ -21,7 +21,8 @@
 /obj/machinery/atmospherics/pipe/Initialize(mapload)
 	. = ..()
 
-	AddElement(/datum/element/undertile, TRAIT_T_RAY_VISIBLE)
+	if(hide)
+		AddElement(/datum/element/undertile, TRAIT_T_RAY_VISIBLE) //if changing this, change the subtypes RemoveElements too, because thats how bespoke works
```

Comment:
so wait a sec here

this never gets added for manually constructed pipes if they get covered up?

Reply:
Yup, same way it used to be. I could make some weird exceptions, but with the way it works I’d have to do a lot of fucky stuff

Reply:
I somehow haven’t actually tested that case with default pipes, but I see no reason why not. Default pipes have hide set to TRUE, so the element gets added and they should be  coverable

Reply:
In another place, what if you place pipe ower cowered tile?

Reply:
Then they're exposed, until someone removes the plating and then covers it. Exactly the way they're intended

Reply:
Not intended. Pipes wrenched over floor must keep be over floor.

Reply:
Really? Huh.